autodocodec/src/Autodocodec/Class.hs

```diff
@@ -108,6 +109,10 @@ instance HasCodec Word64 where
 instance HasCodec JSON.Value where
   codec = ValueCodec
 
+instance HasCodec a => HasCodec (Identity a) where
+  codec = dimapCodec runIdentity Identity codec
+  listCodecForStringCompatibility = dimapCodec (runIdentity <$>) (Identity <$>) listCodecForStringCompatibility
```

Comment:
*shrugs* I just figured this would be the best way to ensure that wrapping with `Identity` kept the same behaviour of the underlying type.

Figured if the base type implemented `listCodecForStringCompatibility` explicitly then using the default for the `Identity` wrapper might change behaviour.